app/controllers/Common.scala

```diff
@@ -3,7 +3,7 @@ package controllers
 import play.api.mvc._
 
 trait Common {
-  self: Controller =>
+  self: BaseController =>
```

Comment:
We are using the `NotFound` method. But this could be replaced by a direct call to `Results` object.